### DISCUSSION TITLE
notes/editor/

### DISCUSSION THREAD

**Discussion by giscus** [SAST-skill-docers/sast-skill-docs#63] [Announcements]
# notes/editor/

https://docs.net9.org/notes/editor/

**Comment by AlbertZhaoCA** [comment]
The section on [article stance](https://docs.net9.org/notes/editor/#_5) inspired me to edit my organization's leap manual. It's important to write tutorials from the perspective of beginners, without assuming that the audience already has certain knowledge and providing references that can be easily access. Love from WKU 😘, it helps us a lot.
